playbooks/terraform-provider-huaweicloud-acceptance-test-fusioncloud/run.yaml

```diff
@@ -58,11 +58,11 @@
 
           # Run except the DNS/FW/LB test 100 testcases at a time
           testcases=`go test ./huaweicloud/ -v -list 'Acc'`
-          testcases=`echo "$testcases" | sed '$d' | grep -v -e IAM -e Kms -e Rds -e RDS -e S3 -e SFS -e SMN -e DNS -e ELB -e Nat -e TestAccProvider`
+          testcases=`echo "$testcases" | sed '$d' | grep -v -e Kms -e Rds -e RDS -e S3 -e SFS -e SMN -e DNS -e ELB -e LB -e Nat -e Images`
           # Add OS_DEBUG=1 TF_LOG=debug flags for debuging
           mkdir -p "{{ ansible_user_dir }}/workspace/test_results/"
```

Comment:
need also to run images related tests separately without OS_DEBUG=1 ?

Reply:
Thank your testing, I will update PR.